tinywallet/tinywallet/screens.py

```diff
@@ -810,27 +811,91 @@ def restoreClicked(self):
         """
         User has selected to generate a wallet from a mnemonic seed.
         """
-        restoreScreen = MnemonicRestorer(self.app)
-        self.app.appWindow.stack(restoreScreen)
+        restoreScreen = MnemonicRestorer()
+        app.appWindow.stack(restoreScreen)
+
+
+class AssetScreen(Screen):
+    """
+    AssetScreen is screen for choosing one account out of many, or changing
+    asset-level settings.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.isPoppable = False
+        self.canGoHome = False
+        self.wgt.setFixedSize(
+            TinyDialog.maxWidth * 0.8,
+            TinyDialog.maxHeight * 0.8 - TinyDialog.topMenuHeight,
+        )
+        self.accountScreens = {}
+
+        logo = QtWidgets.QLabel()
+        logo.setPixmap(pixmapFromSvg(DCR.LOGO, 25, 25))
+        lbl = Q.makeLabel("Decred", 25, fontFamily="Roboto Medium")
+        gear = QtWidgets.QLabel()
+        gear.setPixmap(pixmapFromSvg("gear.svg", 20, 20))
+        # Gear icon will lead to an asset settings screen. Hide for now.
+        gear.setVisible(False)
```

Comment:
This will open a settings screen in the next PR. I was going to do it here, but I thought the scope was already getting a little big. Hiding it for now.